app/controllers/api/v1/lists_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::V1::ListsController < Api::ApiController
+  respond_to :json
+
+  before_action :authenticate
+
+  def index
+    render json: @user.lists
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def create
+    @list = @user.lists.new(list_params)
+    if @list.save
+      render json: @list
+    end
+  end
+
+  def update
+    respond_with List.find(params[:id]).update_attributes(list_params)
```

Comment:
Scope `find` query on `@user.lists` instead of `List` to prevent a logged in attacker updating the title of a victim's list.